koku/api/report/rate/serializers.py

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
Why put these files under `koku/api/report`?
Would prefer `koku/api/rate` for this as its not really the report api.